src/main/scala/nl/knaw/dans/easy/bag2deposit/ddm/ReportRewriteRule.scala

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (C) 2020 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.bag2deposit.ddm
+
+import better.files.File
+import nl.knaw.dans.easy.bag2deposit.parseCsv
+import nl.knaw.dans.lib.logging.DebugEnhancedLogging
+
+import scala.xml.transform.RewriteRule
+import scala.xml.{ Elem, Node }
+
+case class ReportRewriteRule(cfgDir: File) extends RewriteRule with DebugEnhancedLogging {
+
+  case class ReportCfg(uuid: String, label: String, regexp: String)
+
+  private val digit = "[0-9]"
+
+  /** alpha numeric (and a little more) */
+  private val an = "[-_/.a-z0-9]"
+
+  /** just one that does not match easy-dataset:99840 "Arcadis Archeologische Rapporten [2017 - 116]" */
+  val nrRegexp = s"\\W+$an*$digit$an*"
+
+  private val trailer = "([.]|:.*)?"
+  val nrTailRegexp = s"$nrRegexp$trailer"
+  private val missedRegExp = s".*(notitie|rapport|bericht|publicat).*$nrRegexp$trailer"
+
+  val reportMap: Seq[ReportCfg] = parseCsv(cfgDir / "ABR-reports.csv", 0)
+    .map(r => ReportCfg(
+      uuid = r.get(0),
+      label = r.get(1),
+      regexp = r.get(2).trim + nrTailRegexp,
```

Comment:
@janvanmansum 
You don't want column numbers but names. Our code base has only one [example](https://github.com/DANS-KNAW/easy-springfield/blob/104c1511047bdc09b8c2669b83a1c934fbca47ca/src/main/scala/nl.knaw.dans.easy.springfield/CreateSpringfieldActions.scala#L41-L43). To be consistent I would als have to apply it to the other CSV files. That seems to conflict with the provenance in their headers:
https://github.com/DANS-KNAW/easy-convert-bag-to-deposit/blob/0226a00019c349824af839541d41ac2c73de1d82/src/test/resources/debug-config/ABR-complex.csv#L1-L2
https://github.com/DANS-KNAW/easy-convert-bag-to-deposit/blob/0226a00019c349824af839541d41ac2c73de1d82/src/test/resources/debug-config/ABR-period.csv#L1-L2

Is it worth the trouble?